packages/performance-tests/test/TransformerRegression.test.ts

```diff
@@ -33,15 +34,15 @@ const testCasesMap = new Map([
 const outputDir = path.join(__dirname, ".output");
 
 const setupTestData = async () => {
-  const logLevel = process.env.LOG_LEVEL ? Number(process.env.LOG_LEVEL) : LogLevel.Warning;
+  const logLevel = process.env.LOG_LEVEL ? Number(process.env.LOG_LEVEL) : LogLevel.Error;
 
   assert(LogLevel[logLevel] !== undefined, "unknown log level");
 
   Logger.initializeToConsole();
-  Logger.setLevelDefault(logLevel);
-  Logger.setLevel(TransformerLoggerCategory.IModelExporter, LogLevel.Warning);
-  Logger.setLevel(TransformerLoggerCategory.IModelImporter, LogLevel.Warning);
-  Logger.setLevel(TransformerLoggerCategory.IModelTransformer, LogLevel.Warning);
+  Logger.setLevelDefault(LogLevel.Error);
```

Comment:
the LOG_LEVEL env variable should affect the default log level, not the log level of the transformer only. Add another variable if you want to configure that separately.

Reply:
done